Documentation/ContentObjects/Pageview/Index.rst

```diff
@@ -0,0 +1,435 @@
+..  include:: /Includes.rst.txt
+..  index:: Content objects; PAGEVIEW
+..  _cobj-pageview:
+
+========
+PAGEVIEW
+========
+
+..  versionadded:: 13.1
+
+    This API is considered experimental until TYPO3 v13 LTS. It will be further
+    adapted to be used for further rendering of
+    custom Content Elements and Content Blocks.
```

Comment:
Do we already maybe have link targets? Should we link to ext:contenr-blocks for the time
being?

Reply:
I think this is a very temporary note that will be removed as soon as the PAGEVIEW is not experimental anymore?